apps/src/javalab/constants.js

```diff
@@ -0,0 +1,5 @@
+export const csaViewMode = {
```

Comment:
nit: use capital-casing/all-caps for enums/values (e.g., `CsaViewMode.NEIGHBORHOOD`). we also have a [`makeEnum` helper](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/utils.js#L447) that you could use here